inference/trillium/JetStream-Maxtext/Llama2-7B/README.md

```diff
@@ -71,7 +71,7 @@ export WEIGHT_DTYPE=bfloat16
 export PER_DEVICE_BATCH_SIZE=11
 
 cd ~/maxtext
-python MaxText/maxengine_server.py \
+python -m MaxText.maxengine_server \
```

Comment:
It would be good to checkout a specific MaxText commit so that future MaxText changes don't break the recipes. [Example here](https://github.com/AI-Hypercomputer/tpu-recipes/tree/main/training/trillium/Llama3-8B-MaxText/v6e-8#install-maxtext-and-build-docker-image)

Reply:
guess we want to run `git checkout tpu-recipes-v0.1.4` before `python -m MaxText....`